### ISSUE TITLE
provide builds for oneplus 7 pro

### ISSUE THREAD

**Issue by ethael** [lineageos4microg/docker-lineage-cicd#61]
Hello,

lineageos supports oneplus 7 pro (guacamole).
can you add it to your supported device build list please? thanks a lot.

https://wiki.lineageos.org/devices/guacamole

**Comment by benwaffle** [comment]
The code name is `guacamole` - https://github.com/LineageOS/hudson/blob/master/lineage-build-targets#L43

**Comment by thequint-lab** [comment]
While there are no official builds yet, I am successfully using an unofficial build over on [XDA](https://forum.xda-developers.com/oneplus-7-pro/development/rom-lineageos-16-microg-support-t3960116)

I would love to see official builds soon, at least for the sake of regular OTAs.

**Comment by thinkpadder1** [comment]
@corna Do you need help with this? It would be nice to see the build error log publicly (or if you can copy/paste it here). I'd like to dive in, but don't really want to set up the environment if it's a major upstream issue.

In the meanwhile, I'll attempt to build guacamole using https://github.com/fkoester/lineageos-microg-buildenv

**Comment by tliebeck** [comment]
Would love to see this as an official build.  I built it myself using this docker image, have been running for a week with no issues and absolutely love it.  Donated.  I don't mind building myself but imagine a lot of folks would benefit having a downloadable version.

**Comment by BoBeR182** [comment]
Is there a reason this is not being implemented, how can we help with any roadblocks?

**Comment by prg318** [comment]
Would love to see this happen

**Comment by loupianche** [comment]
Same for me, getting microg working with stock lineageos is a pain

**Comment by jorisvandijk** [comment]
I am also very interested in this. They state on the website that all Lineageos supported devices are also supported by them. Does anyone know why the OnePlus 7 pro is excluded?

**Comment by byronmansfield** [comment]
I'm also interested in this. I'm trying to build it using the docker method, though I keep running into issues. I'm making progress though. Anyone else have any luck?

**Comment by thequint-lab** [comment]
Since lineageos4microg seems to be dead judging by the activities in the past 6 months, I stumbled upon a ROM integrated with MicroG and automatic updates, OmniROM has just released such a rom [here](https://dl.omnirom.org/oneplus7pro/). I haven't tested it though, but I thought I'd share for those interested in pre-built builds with microG for the 7 Pro.

**Comment by byronmansfield** [comment]
Interesting. Seems promising. Nice find. Thanks for mentioning it here. I will research this further.

By chance, do you happen to know where I might find more information about how it differs from LineageOS in regards to privacy and other features? I see in their website screenshots it looks like it ships with google maps. I guess I'm curious if they swapping out the geolocator and NLP. Which I suppose is more on the microg side, so if it is supported then it should be good. But I don't see a lot of documentation around what exactly these details are for OmniROM.

In case the rest of the group may be interested. Another one I've had my eye on which seems promising, is [https://e.foundation/](https://e.foundation/). Though it doesn't seem like they are in full swing yet. It may still be early adoption if you want to try it out. They do say they support OP 7 Pro though. [https://doc.e.foundation/devices/guacamole/](https://doc.e.foundation/devices/guacamole/)

**Comment by jorisvandijk** [comment]
@byronmansfield , I've come across /e/ before, though this was a long while ago. They didn't have any products ready at the time. This might be a great solution if our device doesn't get official support here. Did you by any chance test the ROM? It mentions in the issues that it does not have carrier support which obviously is a deal breaker.

**Comment by bellegarde-c** [comment]
You will find here my custom LineageOS build for Oneplus 7 pro with MicroG:

https://gitlab.com/gnumdk/lineage-guacamole

OTA supported.

**Comment by BoBeR182** [comment]
Why was this closed, there is no proper resolution, a 3rd party repo with custom signing keys is not the same as download.lineage.microg.org

**Comment by spolack** [comment]
Seems my comment got lost. LineageOS 17.1 support is given, #89 remains open until the first build process went through. 

TL;DR: LineageOS 17.1 Builds are queued after current LineageOS 16.0 cycle.

**Comment by thequint-lab** [comment]
First build is out: https://download.lineage.microg.org/guacamole/

Appreciate the efforts @spolack and team!

**Comment by r4nd0wn** [comment]
Is there any reason why I can find a guacamole build and guacamoleb build?

**Comment by Baschdl578** [comment]
They are here:
https://download.lineage.microg.org/guacamole/
Ist that folder empty for you?
